### PR TITLE
Send donations donated to milestone owner if recpient was campai

### PR DESCRIPTION
**Pull request by mohammadranjbarz**
Send donations collected to milestone owner if recipient was campaign or anonymous user

related to #315
related #347

### CODE REVIEW

File: src/blockchain/milestones.js [hunk 1]
```diff
@@ -34,8 +34,13 @@ const getDonationPaymentsByToken = donations => {
 const createPaymentConversationAndSendEmail = async ({ app, milestone, txHash }) => {
   try {
     const milestoneId = milestone._id;
-    const { recipient, campaignId, title } = milestone;
+    const { recipient, campaignId, title, owner } = milestone;
 
+    // When  recipient is an address that doesnt have account in Giveth recipient is null
+    // so below checking is for this purpose to send emails for milestone's owner instead of recipient
+    const email = (recipient && recipient.email) || owner.email;
```
Comment (aminlatifi) [inline src/blockchain/milestones.js:41]: As #315 says the requirement is the campaign owner (not milestone owner) should receive email when the milestone recipient is the campaign.

**Review by aminlatifi — CHANGES_REQUESTED** [review]

File: src/blockchain/milestones.js (hunk 1 above)
Comment (mohammadranjbarz) [reply src/blockchain/milestones.js:41]: Yes Amin I was wrong
For campaign owner the code doesn't come  here, it's just for milestones that have anonymous user recipient

Comment (aminlatifi) [reply src/blockchain/milestones.js:41]: @mohammadranjbarz I am not sure we should email to milestone owner, or if we do should do always (not only when recipient is anonymous)
@horus2021 what do you think?

Comment (mohammadranjbarz) [reply src/blockchain/milestones.js]: @aminlatifi Now I undo sending email for owner if recipient is null

**Pull request merged**